apps/condo/domains/contact/access/Contact.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Generated by `createschema contact.Contact 'property:Relationship:Property:SET_NULL; name:Text; phone:Text; unitName?:Text; email?:Text;'`
+ */
+
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+const get = require('lodash/get')
+const { Contact } = require('../utils/serverSchema')
+
+async function canReadContacts ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    return {
+        organization: { employees_some: { user: { id: user.id }, isBlocked: false } },
+    }
+}
+
+async function canManageContacts ({ authentication: { item: user }, originalInput, operation, itemId, context }) {
+    if (!user) return false
+    if (user.isAdmin) return true
+    if (operation === 'create') {
+        const organizationId = get(originalInput, ['organization', 'connect', 'id'])
+        const canManageContacts = await checkOrganizationPermission(user.id, organizationId, 'canManageContacts')
```

Comment:
can access to property?

Reply:
Yes, it can.
`canReadProperties` has similar logic: should be employed in the organization in question and should not be blocked.